### PR TITLE
Add 2D histograms for pt jpsi vs pt daughter

### PR DESCRIPTION
**Pull request by zhangbiao-phy**

### CODE REVIEW

File: Analysis/Tasks/PWGHF/taskJpsi.cxx [hunk 1]
```diff
@@ -188,6 +191,16 @@ struct TaskJpsiMC {
...
         registry.fill(HIST("hPtGen"), particle.pt());
         registry.fill(HIST("hEtaGen"), particle.eta());
+        registry.fill(HIST("hYGen"), RecoDecay::Y(array{particle.px(), particle.py(), particle.pz()}, RecoDecay::getMassPDG(particle.pdgCode())), particle.pt());
+
+        float ptProngs[2];
+        int counter = 0;
+        for (int iD = particle.daughter0(); iD <= particle.daughter1(); ++iD) {
+          ptProngs[counter] = particlesMC.iteratorAt(iD).pt();
+          counter++;
+        }
```
Comment (vkucera) [inline Analysis/Tasks/PWGHF/taskJpsi.cxx]: What's the point of doing this?

Comment (zhangbiao-phy) [reply Analysis/Tasks/PWGHF/taskJpsi.cxx]: here I reference "taskX.cxx" to assign the pt of daughters

Comment (vkucera) [reply Analysis/Tasks/PWGHF/taskJpsi.cxx]: * It is not in the `dev` version, which means it has not been reviewed.
* I don't see why you want to use an array for storing two `float` values which you use just once.
* Why don't you fill them directly?

Comment (zhangbiao-phy) [reply Analysis/Tasks/PWGHF/taskJpsi.cxx]: Ok, I can simply this

**Review by vkucera — APPROVED** [review]

**Pull request merged**